content/typescript/concepts/arrays/arrays.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Arrays'
-Description: 'Arrays in TypeScript are considered to be "generic" across a single type of value. Once an array is known to contain a particular type of value, only that type is allowed in the array. Array types can be inferred from initial values in the array. In this example, the vowels array is inferred to consist of strings, and will allow pushing only strings: ts const vowels = [a, e, i, o, u]; vowels.push(y); // Ok'
+Description: 'In TypeScript, arrays are considered to be collections of single, "generic" types of values. All elements in an array must be of the same type of data.'
```

Comment:
Is that last sentence fully correct? An array could have multiple types if you use `const myArray:(string|number)[]`. Could the sentence explain that the array can only contain the types specified in the declaration rather than only one type?